src/main/java/Ui.java

```diff
@@ -1,25 +1,25 @@
 import java.util.Scanner;
 
 class Ui {
-    private final String LOGO = " ____        _        \n"
+    private final String logo = " ____        _        \n"
             + "|  _ \\ _   _| | _____ \n"
             + "| | | | | | | |/ / _ \\\n"
             + "| |_| | |_| |   <  __/\n"
             + "|____/ \\__,_|_|\\_\\___|\n";
-    private final String LINE = "______________________________";
+    private final String line = "______________________________";
```

Comment:
`line` can be marked `static` and `final`, keeping the variable name uppercased.